frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -39,11 +39,13 @@ const QuickLinksAndSettingsBtn = ({ linkGroups }) => {
         <Grid item>
           <QuickLinksCard linkGroups={linkGroups} />
         </Grid>
-        <Grid item style={{ padding: "10px" }}>
-          <IconButton onClick={() => {}}>
-            <SettingsIcon />
-          </IconButton>
-        </Grid>
+        <SimpleFeatureFlag feature="projectSettings">
```

Comment:
Can we rename this to projectCatalogSettings? That way its inline with the projectCatalog feature flag